<?xml version="1.0"?>
<launch>

  <node name="r_arm_wrench_estimation" pkg="pr2_wrench_estimation" type="pr2_wrench_estimation" cwd="node" respawn="false" output="screen">
    <rosparam command="load" file="$(find pr2_wrench_estimation)/config/right_arm_wrench_estimation.yaml"/>
  <remap from="r_arm_wrench_estimation/controller_state" to="/r_arm_vel/state"/>
  </node>

</launch>